src/SettingsTab.ts

```diff
@@ -64,5 +64,25 @@ export class SettingsTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
             });
+
+        containerEl.createEl('div', {
+            cls: 'setting-item-description',
+            text: 'Enable reminder on edit/create task (requires obsidian-reminder plugin)',
+        });
+
+        new Setting(containerEl)
+            .setName('Enable reminder on edit/creat task')
+            .setDesc(
+                'Enable reminder on edit/create task (requires obsidian-reminder plugin)',
```

Comment:
There seems to be some duplicated text here. I may refactor it out to create a variable for the duplicated text.